src/scvi/model/_scvi.py

```diff
@@ -134,6 +148,7 @@ def __init__(
             f"dropout_rate: {dropout_rate}, dispersion: {dispersion}, "
             f"gene_likelihood: {gene_likelihood}, latent_distribution: {latent_distribution}."
         )
+        self._module_init_on_train = False
 
         if self._module_init_on_train:
```

Comment:
remove module_init_on_train. This is deprecated code with the new dataloader implementation.